modules/candidate_list/test/candidate_listTest.php

```diff
@@ -182,9 +182,9 @@ function testFilters()
                "return document.querySelector('$btn').click()"
            );
            //female
-           $this-> _testFilter(self::$sex, "20 rows displayed of 334", '1');
+           $this-> _testFilter(self::$sex, "20 rows displayed of 228", '1');
            // male
-           $this-> _testFilter(self::$sex, "20 rows displayed of 328", '2');
+           $this-> _testFilter(self::$sex, "20 rows displayed of 210", '2');
```

Comment:
```suggestion
           $this-> _testFilter(self::$sex, "20 rows displayed of ", '2');
```

I don't think the maximum number of candidates matters for this test. It might help.

Reply:
i think the intent of the test is to check the "correct" number of candidates get returned....
ur solution bypasses that BUT I'M VERY TEMPTED TO DO IT

Reply:
Is 210 the number of Male candidates when the Project filter is applied when you test locally? I think that's what it's trying to do

Reply:
because it seems that travis tests the PR on the current RB dataset on the major branch, with the different PRs being merged by the time I rebase the PR to determine the numbers, another PR gets merged and modifies the raisinbread data and changes the number.

This test is particularly annoying, if any of the asserts fail, the remaining ones dont get run so if they will also fail they fail only after the first assert gets fixed and the PR is submitted to travis again. so if all the asserts are gonna fail, you'll need to run it 10 times through travis... gets what happens meanwhile, another PR gets merged and the numbers change again